bench/ArrayBench.java

```diff
@@ -0,0 +1,30 @@
+package benchmark;
+
+import com.sun.cldchi.jvm.JVM;
+
+public class ArrayBench {
+    void runBenchmark() {
+        long start, time;
+        int len = 1;
+        int iterations = 1000;
+
+        System.out.println("Time to allocate int arrays of various lengths " + iterations + " times, by length, in ms:");
+
+        for (int i = 0; i < 20; i++) {
+
+            start = JVM.monotonicTimeMillis();
+            for (int j = 0; j < iterations; j++) {
+                int[] array = new int[len];
```

Comment:
> Nit: It would be good if we could format the output of the benchmark in a way that is simple to parse, for example:

Ok, I made it output this way in a5800de.